### ISSUE TITLE
Where can we download the pre-trained model used to generate feature-embeddings for the images?

### ISSUE THREAD

**Issue by LIU-YUXI** [fdbtrs/Self-restrained-Triplet-Loss#10]
Where can I download the model defined in face_model.py to preprocess the embedding of the input images? It seems that only the weights of fine-tuned cnn models are available for download in the repository. Thank you.
